tests/src/Interop/Variant/VariantTest.cs

```diff
@@ -0,0 +1,360 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+using TestLibrary;
+
+class Test
+{
+    private const string NativeLibrary = "VariantNative";
+
+    private const byte NumericValue = 15;
```

Comment:
Yep. Here is my gripe. Now the answer is in two places instead of one. Tests should be written with 1 source of truth.